docs/design/auth.md

```diff
@@ -61,3 +61,9 @@ NOTE: The point is to make the token not *readily* available to developer contri
 The token is meant for tasks and scripts that are trusted by the build admin by (1) installing the task or (2) directing a build definition to run that script.  
 The goal is to avoid having the token accidentally leak in scripts.  
 Even then, the token will expire at the end of the job which helps mitigate any accidental exposure.
+
+## Keeping RSA private key - security implications for 'run once' agent setup
+Agent stores RSA private key in a file on disk. For Windows host - it is protected by DPAPI; on Linux/OSX - it sets up chmod permissions (read & write) to the owner only (by calling 'chmod 600' command).
+
+An RSA private key is being created during agent configuration and removed only after running of 'config remove' command.
+For 'run once' agent setup - it keeps the private key file the same way, so it remains on disk after agent being shut down. If you don't want this file to be available for the user you configure agent with - make sure that you removed it by your own.
```

Comment:
in markdown, each sentence should be on its own line. 

The second sentence is unclear. 

for which user do we chmod the file? The one configuring or the one running? (probably the one configuring)
If they delete the file, what happens? how would they recover? 

For run once, I think there is one config and multiple runs, so the .creds file would be important for future runs.

Reply:
Thanks! Amended changes per comment, added details about users and rsa file removal case.